Makefile

```diff
@@ -107,17 +107,8 @@ prettier: ## prettify the source code using prettier
 test: build test-unit lint test-e2e ## launch all tests
 
 test-unit: ## launch unit tests
-	@if [ "$(CI)" != "true" ]; then \
-		echo "Running unit tests..."; \
-		yarn -s test-unit; \
-	fi
-	@if [ "$(CI)" = "true" ]; then \
-		echo "Running unit tests in CI..."; \
-		yarn -s test-unit-ci; \
-	fi
-
-test-unit-watch: ## launch unit tests and watch for changes
```

Comment:
why did you remove this one?

Reply:
See README.

Reply:
Nvm I'll revert that. Jest does not watch automatically when not on CI. That's done by react-scripts.